assets/hdfs_reader/index.js

```diff
@@ -0,0 +1,239 @@
+'use strict';
+
+const Promise = require('bluebird');
+// const path = require('path');
+const Queue = require('@terascope/queue');
+// const fs = require('fs');
+
+
+function getClient(context, config, type) {
```

Comment:
Same issue with this. At a minimum these should be pulled into a common lib but I believe a function doing this is available as a Teraslice API.